readme.md

```diff
@@ -62,9 +62,11 @@ _Setting up a WebDAV server_
 
 _Standalone servers and personal cloud solutions specifically built for WebDAV and its extensions_
 
+https://sabre.io/baikal/
```

Comment:
This seems incorrectly formatted, can you fix?

Reply:
sorry, wanted to add it but when checked it was only CalDAV